tools/test_containers_compose

```diff
@@ -5,7 +5,7 @@ set -euo pipefail
 for i in webui worker; do
   (
     cd container/$i/ &&
-    sudo docker-compose build -q --parallel &&
+    sudo docker-compose build --parallel &&
```

Comment:
Where do I see that this worked? The tests pass and I'm not sure where to find the change

Reply:
Just look at the very verbose output within the CI pipeline compose / webui-docker-compose. It now shows all the zypper output which is likely what we're interested in.